builtin_json.go

```diff
@@ -334,7 +334,15 @@ func (ctx *_builtinJSON_stringifyContext) str(key Value, holder *Object) bool {
 			ctx.buf.WriteString("false")
 		}
 	case valueString:
-		ctx.quote(value1)
+		if ascii, ok := value1.(asciiString); ok {
+			j, err := json.Marshal(ascii)
+			if err != nil {
+				panic(err)
```

Comment:
Should this be `panic(r.newError(r.global.SyntaxError, err.Error()))`?

Reply:
I was following the other patterns in this file which just did panic(err). I don't see a world where this could happen so we should be good